.github/workflows/build-images.yml

```diff
@@ -0,0 +1,95 @@
+name: Build images
+
+on:
+  - push
+
+jobs:
+  cc7_lcg95:
+    runs-on: ubuntu-latest
+    needs: base_images
+    steps:
+      - uses: actions/checkout@v1
+      - name: Extract branch name
+        shell: bash
+        run: echo "::set-env name=branch::$(echo ${GITHUB_REF#refs/heads/} | sed 's/\//_/g')"
+      - uses: acts-project/Publish-Docker-Github-Action@master
+        with:
+          registry: docker.pkg.github.com
+          name: acts-project/machines/cc7_lcg95
```

Comment:
In principle yes. It won't push to the upstream in any case from a fork, because of missing permissions. I'd like to keep this for now though, since I would keep this repo off of the fork workflow for now.

Reply:
Ok. I might play around with this on my fork, though.